Calculator.sln

```diff
@@ -15,6 +17,10 @@ Global
 		{42B5D5A2-97DE-41F7-A20D-FD68240F701E}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{42B5D5A2-97DE-41F7-A20D-FD68240F701E}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{42B5D5A2-97DE-41F7-A20D-FD68240F701E}.Release|Any CPU.Build.0 = Release|Any CPU
+		{1CB150D8-39B5-46D5-B16A-42B97E05C7AC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{1CB150D8-39B5-46D5-B16A-42B97E05C7AC}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{1CB150D8-39B5-46D5-B16A-42B97E05C7AC}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{1CB150D8-39B5-46D5-B16A-42B97E05C7AC}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
There are some inconsistencies with indent sizes through the different files. Here it looks like you're using tabs, in other files you're using 4 spaces and in another you're using 2 spaces. You can change your default tab spacing within your editor. For this project I think I prefer 4 spaces for an indent so if you could change this file and `Program.cs` to be consistent :D